### PR TITLE
Adds additional Star Atlas poster NFT mints

### PR DESCRIPTION
**Pull request by MikeLuDev**
- [x] I will not ping the Discord about this pull request.

**Please provide the following information for your token.**

Please include change to the src/lib/<env>.json and in the PR

At minimum each entry should have
* Token Address: 
* Token Name: 
* Token Symbol: 
* Logo
* Link to the official homepage of token:
* Coingecko ID if available (https://www.coingecko.com/api/documentations/v3#/coins/get_coins__id_):

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solana/spl-token-registry",
-  "version": "0.2.81",
```
Comment (bartosz-lipinski) [inline package.json:3]: you dont need to update version that happens automatically.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### 0.2.82 (2021-05-11)
```
Comment (bartosz-lipinski) [inline CHANGELOG.md:5]: remove this change

**Review by bartosz-lipinski — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (MikeLuDev) [reply package.json:3]: Not sure how this got in here - the commit `fe93904` was added after I opened the PR 🤔

Comment (MikeLuDev) [reply package.json]: Ah I see - the GH workflow ran automatically in my forked repo. Disabled them.

**Pull request merged**